packages/commons/src/package-lock.json

```diff
@@ -1,25 +1,11 @@
 {
-  "name": "src",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
-    "": {
-      "license": "MIT",
-      "dependencies": {
-        "axe-core": "^4.4.3"
-      }
-    },
-    "node_modules/axe-core": {
-      "version": "4.4.3",
-      "license": "MPL-2.0",
-      "engines": {
-        "node": ">=4"
-      }
-    }
-  },
+  "lockfileVersion": 1,
```

Comment:
This doesn't seem related to the other changes in this PR; was including it intentional? (the change suggests that you've built with an old version (<7) of NPM)

Reply:
Reverted the changes by deleting the `package-lock.json` and re-running `npm install`